Embedded/MaxMix/Config.h

```diff
@@ -153,4 +153,4 @@ static const uint8_t DISPLAY_GAME_EDIT_CHAR_MAX_WIDTH = DISPLAY_GAME_EDIT_CHAR_M
 static const uint8_t DISPLAY_GAME_WIDGET_VOLUMEBAR_HEIGHT = 7;
 static const uint8_t DISPLAY_GAME_VOLUMEBAR_WIDTH = DISPLAY_AREA_CENTER_WIDTH - DISPLAY_GAME_EDIT_CHAR_MAX_WIDTH - DISPLAY_MARGIN_X2 - 2 - DISPLAY_MARGIN_X1 * 2;
 
-static const uint32_t resetAfterInactivity = 4000;
+static const uint32_t DEVICE_RESET_AFTER_INACTIVTY = 10000;
```

Comment:
I don't think increasing the timeout is going to fix anything. 
People that have reported that the device gets disconnected under some circumstances, such as having their CPU run at 100%.  I was not able to reproduce that at all so there is something else going on with their PCs. But regardless, if their CPU runs at 100% for 10 seconds they will have the same issue. So increasing the time is not really fixing the problem.
As a side effect, now the disconnect detection is not very useful if it takes 10 seconds for the device to realize that the connection has been lost.

Reply:
Would you be ok with 5s ?

Reply:
Makes sense!

Reply:
fixed